CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+### Thanks, for considering to contribute to this repository! 
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+If you are looking forward to start contributing to beautiful world of open source.
+**Start it right now.**
+
+The basic guidelines for contributing are as follows:  
+1. Fork the repository.
+2. Clone repository in your local environment.
```

Comment:
Can you add 1 more step here something like this
`(Options) Give star to the repo if you like the project`